package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think this file was added by accident - we have no need of a package-lock.json at project root. Can you remove it please?